packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -207,32 +240,40 @@ export const styles = css`
         css`
             .control {
                 background-color: rgba(${borderRgbPartialColor}, 0.1);
-                border-color: transparent;
             }
 
             .control:hover {
                 background-color: transparent;
-                border-color: ${borderHoverColor};
             }
 
             .control${focusVisible} {
+                background-color: transparent;
+            }
+
+            .control::before:hover {
                 background-color: rgba(${borderRgbPartialColor}, 0.1);
-                border-color: ${borderHoverColor};
+                padding: 2px;
             }
 
-            .control${focusVisible}:hover {
-                background-color: transparent;
+            .control::before${focusVisible} {
+                background-color: rgba(${borderRgbPartialColor}, 0.1);
+                padding: 2px;
             }
 
-            .control:active {
+            .control::before:active {
                 background-color: ${fillSelectedColor};
-                border-color: ${fillSelectedColor};
+                padding: 1px;
             }
 
             :host([disabled]) .control {
                 background-color: rgba(${borderRgbPartialColor}, 0.1);
                 border-color: transparent;
             }
```

Comment:
The selection target of this element is `.control` and should be grouped with the other `.control` targetted selectors.